database/migrations/2023_02_26_112116_split_form_and_payment_completion_emails.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('travel', static function (Blueprint $table): void {
+            $table->dropColumn('completion_email_sent');
+            $table->boolean('payment_completion_email_sent')->default(false);
+            $table->boolean('form_completion_email_sent')->default(false);
```

Comment:
This looks like it will migrate data incorrectly for existing travel assignments. I would naively suggest setting both new fields to true if `payment_completion_email_sent` is true currently

Reply:
I don't think it needs to be that precise. There isn't any way to trigger this email without modifying a TA, which can't happen if it's complete.